boofuzz/sample.py

```
@@ -0,0 +1,47 @@
+    """
+    Función que se ejecuta cuando el servidor se cae.
+    Guarda información sobre el crash.
+    """
+    fuzz_data_logger.log_fail("💥 Servidor CRASHEADO 💥")
+    with open("crash_log.txt", "a") as f:
+        f.write(f"Crash detectado en intento {session.num_mutations}\n")
+
+# Configurar el objetivo
+target_ip = "127.0.0.1"
+target_port = 1337
+target = Target(connection=TCPSocketConnection(target_ip, target_port))  # Se usa TCPSocketConnection en lugar de SocketConnection
```

Ruff (0.8.2)

`Target` may be undefined, or defined from star imports

(F405)